web/sites/default/settings.php

```diff
@@ -75,6 +75,12 @@
     $config['environment_indicator.indicator']['fg_color'] = '#ffffff';
     $config['environment_indicator.indicator']['name'] = 'Dev';
   }
+  elseif ($_ENV['PANTHEON_ENVIRONMENT'] === 'sandbox') {
+    $primary_domain = 'sandbox.portland.gov';
+    $config['environment_indicator.indicator']['bg_color'] = '#3455eb';
+    $config['environment_indicator.indicator']['fg_color'] = '#ffffff';
+    $config['environment_indicator.indicator']['name'] = 'Dev';
```

Comment:
You should probably change the name to 'Sandbox'

Reply:
Thanks! I've pushed another update. Also added "only run sandbox test on sandbox" as your PE-524.